packages/perps-exes/assets/config-price.toml

```diff
@@ -2089,3 +2089,75 @@ inverted = false
 [networks.rujira-testnet.ATOM_USDC.feeds-usd.constant]
 price = "1"
 inverted = false
+
+[[networks.rujira-testnet.AVAX_USDC.feeds]]
```

Comment:
Please remove the extra definitions that aren't needed here. Let's keep it to just BTC_USDC for now.